### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by itsrohan2003**
**added my card**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,29 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <div class="card">
```
Comment (TenTraicion) [inline index.html:117]:
```suggestion

       <!--rohan's card start-->
        <div class="card">
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@itsrohan2003, Thanks for the contribution, kindly make the recommended changes and commit.

**Review by itsrohan2003 — COMMENTED** [review]
I have done the requested changes.

**Comment by TenTraicion** [comment]
@itsrohan2003, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**